src/routes/stats/history.rs

```diff
@@ -98,7 +98,7 @@ impl<'a> FromFormValue<'a> for HistoryCursor {
 /// Get the query history according to the specified parameters
 fn get_history(ftl_memory: &FtlMemory, env: &Env, params: HistoryParams) -> Reply {
     // Check if query details are private
-    if FtlConfEntry::PrivacyLevel.read_as::<FtlPrivacyLevel>(&env)? >= FtlPrivacyLevel::Maximum {
+    if FtlConfEntry::PrivacyLevel.read_as::<FtlPrivacyLevel>(env)? >= FtlPrivacyLevel::Maximum {
```

Comment:
This is not a functional change. The reference symbol (`&`) was unnecessary, since `env` is already a reference.